docs/UserGuide.md

```diff
@@ -28,16 +28,18 @@ as well as summary commands.
 
 4. Double-click the file to start the app. The similar GUI shown below (Figure 1) should appear in a few seconds. <br>
    *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*<br>
```

Comment:
```suggestion
   *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*
```

Reply:
Not sure if this is better? Is the white spacing too big for every picture?

Reply:
The white spacing is indeed abit too big with 2 <br>
![image](https://user-images.githubusercontent.com/61489581/97188703-c75ef600-17de-11eb-8813-7ea21a7cffc8.png)
added your change! Thanks!